app/assets/javascripts/application.js.erb

```diff
@@ -597,19 +534,73 @@ onPageLoad(() => {
   });
 
   on('click', 'button.story-preview', (event) => {
-    Lobsters.previewStory(parentSelector(event.target, 'form'));
+    Lobster.previewStory(parentSelector(event.target, 'form'));
   });
 
   // Comment Related Functions
 
+  on('click', 'a.comment-disowner', (event) => {
+    if (confirm("Are you sure you want to disown this comment?")) {
+      let li = parentSelector(event.target, '.comment');
+      fetchWithCSRF('/comments/' + li.getAttribute('data-shortid') + '/disown', {
+        method: 'post'
+      })
+        .then(response => response.text())
```

Comment:
elsewhere you do `response => response.text().then(text => ...`

Is there a reason this one's different?